xml/bura-manual_backup.xml

```diff
@@ -432,10 +403,28 @@ vars:
    </step>
    <step>
     <para>
-     Run the <filename>spread_read_lock.yml</filename> playbook created above.
+     Open a &mariadb; client session on all controllers.
+    </para>
+   </step>
+   <step>
+    <para>
+     Run the command to spread <literal>read lock</literal> on all controllers
+     and keep the &mariadb; session open.
+    </para>
+    <screen>&gt;&gt; FLUSH TABLES WITH READ LOCK;</screen>
+   </step>
+   <step>
+    <para>
+     Open a new terminal and run the
+     <filename>create_db_snapshot.yml</filename> playbook created above.
```

Comment:
Could we put the same description (deleted in line 437-438) here to run playbook.